cmp/ag-grid/AgGridModel.ts

```diff
@@ -574,7 +573,7 @@ export class AgGridModel extends HoistModel {
      */
     setPinnedTopRowData(data: PlainObject[]) {
         this.throwIfNotReady();
-        this.agApi.setPinnedTopRowData(data);
+        this.agApi.setGridOption('pinnedTopRowData', data);
     }
```

Comment:
@cnrudd -- think we missed the matching change for `setPinnedBottomRowData`

Reply:
Fixed. thanks!